lib/api.js

```diff
@@ -0,0 +1,124 @@
+var base      = require('taskcluster-base');
```

Comment:
I guess technically our policy is to do so for every file, which makes a lot of sense in gecko land which is multi license...

From https://www.mozilla.org/MPL/2.0/ "Exhibit A - Source Code Form License Notice" it should be sufficient to just include a LICENSE file, if it's not possible or desirable to put in every file.
So far I've just said it wasn't desirable, because I was well, lazy :)

Anyways, if you feel like adding it to all repos and running tests, go ahead, you'll be a major contributor :)
Note, some of our modules aren't MPL, because they extend apache or MIT licensed code.

Generally, I think stating it in `package.json` and possibly a `LICENSE` file should do.